pageserver/src/tenant/storage_layer/layer_name.rs

```diff
@@ -248,6 +248,17 @@ impl LayerName {
             Image(_) => "image",
         }
     }
+
+    /// Checks if the layer name might be an L0 layer.
+    ///
+    /// **Implementation specific:** an L0 layer is a delta layer and it's key range is from [`Key::MIN`] to [`Key::MAX`]
+    pub fn is_maybe_l0(&self) -> bool {
```

Comment:
I think there is an `is_l0` function somewhere (probably layer map?) and we can move that to a common function

Reply:
Yes there is a `LayerMap::is_l0` function operating on `PersistentLayerDesc`.
https://github.com/neondatabase/neon/blob/3773a945f517a2d451397cc25bd887a710ef8574/pageserver/src/tenant/layer_map.rs#L601-L603

 I will refactor so everyone to take in the key_range.